src/main/kotlin/minesweeper/Blocks.kt

```diff
@@ -0,0 +1,5 @@
+package minesweeper
+
+class Blocks(normalBlocks: List<Block>, mineBlocks: List<Block>) {
+    val blocks: List<Block> = (normalBlocks + mineBlocks).toMutableList().shuffled()
```

Comment:
```suggestion
    val blocks: List<Block> = (normalBlocks + mineBlocks).shuffled()
```
간단하게 나타낼 수 있을 것 같습니다.

Reply:
수정하였습니다